### PR TITLE
Add tiny swabs to RoR shipping views

### PR DESCRIPTION
**Pull request by kschwabefry**
Include HCT tiny swabs identifier sets in shipping views for
reportable conditions, return of results, and sequencing metadata

### CODE REVIEW

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -2093,7 +2093,8 @@ create or replace view shipping.genome_submission_metadata_v1 as
           when identifier_set.name in ('collections-kiosks', 'collections-kiosks-asymptomatic') then 'Shelters'
           when identifier_set.name in ('collections-school-testing-home', 'collections-school-testing-observed') then 'Snohomish Schools'
           when identifier_set.name in ('collections-seattleflu.org') then 'SCH'
-          when identifier_set.name in ('collections-uw-home', 'collections-uw-observed', 'collections-uw-tiny-swabs') then 'HCT'
+          when identifier_set.name in ('collections-uw-home', 'collections-uw-observed', 'collections-uw-tiny-swabs',
+                                       'collections-uw-tiny-swabs-home', 'collections-uw-tiny-swabs-observed') then 'HCT'
```
Comment (davereinhart) [inline schema/deploy/shipping/views.sql:2097]: Do we want to leave the pilot study records `collections-uw-tiny-swabs` here or drop them?

**Review by davereinhart — APPROVED** [review]

File: schema/deploy/shipping/views.sql (hunk 1 above)
Comment (kschwabefry) [reply schema/deploy/shipping/views.sql:2097]: My thinking was to keep them in the genome metadata mapping for completeness. I don't know whether it's likely they would sequence any of the samples from the pilot swabs, but seemed possible

**Comment by kschwabefry** [comment]
Deployed.

**Comment by kschwabefry** [comment]
[Fix Tiny swabs ROR](https://trello.com/c/y6Gd3YQw/1186-fix-tiny-swabs-ror)

**Pull request merged**